src/focus.service.ts

```diff
@@ -362,46 +332,35 @@ export class FocusService {
       return;
     }
 
-    this.rescroll(next, scrollSpeed, this.root);
-
     const attached = selected && isNodeAttached(selected, this.root);
 
-    this.parents = [];
-
     if (attached) {
       // Find the common ancestor of the next and currently selected element.
       // Remove selected classes in the current subtree, and add selected
       // classes in the other subtree. Trigger focus changes on every
       // element that we touch.
-      selected.blur();
-
       const common = getCommonAncestor(next, selected);
-      selected.classList.remove(cssClass.direct);
```

Comment:
How does it work currently? All buttons are given focus already, this just removes the class application which [doesn't seem to be used](https://github.com/WatchBeam/frontend2/search?utf8=%E2%9C%93&q=arc--selected-direct&type=) anywhere.